src/document-register-element.js

```diff
@@ -570,6 +570,11 @@ if (!V0) {
             )
               fragment.appendChild(document.importNode(childNodes[i], !!deep));
             return fragment;
+          case 3:
+            // Handle Text nodes
+            var fragment = document.createDocumentFragment();
+            fragment.textContent = node.wholeText;
```

Comment:
why would the `wholeText` be needed for a `nodeType` 3?

also, any test you can provide to demonstrate the issue and also solve it?

thanks

P.S. I am not sure I understand the issue ... text nodes have no `getAttribute`, so it's not really a responsibility of this library to fix 3rd parts issues, right?

I've never had any with [lighterhtml](https://github.com/WebReflection/lighterhtml#readme) and/or [basicHTML](https://github.com/WebReflection/basichtml#readme), as example.